keps/sig-api-machinery/5073-declarative-validation-with-validation-gen/README.md

```diff
@@ -587,13 +588,13 @@ The below rules are currently implemented or are very similar to an existing val
   </tr>
   <tr>
    <td>
-    numeric limits
+     numeric limits (constant or field refs capable)
    </td>
    <td>
     `+k8s:minimum`, `+k8s:maximum`, `+k8s:exclusiveMinimum`, `+k8s:exclusiveMaximum`
    </td>
    <td>
-    `minimum`, `maximum`, `exclusiveMinimum`, `exclusiveMaximum`
+    minimum, maximum, exclusiveMinimum, exclusiveMaximum (for constants); x-kubernetes-validations (for field refs)
```

Comment:
lost the markdown format. . Intentional?

Reply:
Unintentional, thanks for catching this.  Fixed now.